### PR TITLE
Fix #22: readme update

### LINKED ISSUE #22: README Update
Please confirm the README is updated and accurate for this repository.

### PR DESCRIPTION
**Pull request by vansimke**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,20 +34,13 @@ when available, adding logic when necessary to comply with the more recent parts
...
 ### CSS Style Manipulation
 
-The dom package provides APIs for adding and removing styles *in a stylesheet*, rather than inline.  This allows
+The dom package provides the `addCssRule` function for adding styles *in a stylesheet*, rather than inline.  This allows
 styles to be added programmatically on a per-selector basis, which is often more flexible and efficient than
-setting styles directly inline on one element.
-
-* `dom.addStyleDeclaration` - Creates a `CSSStyleDeclaration`, adds it to a stylesheet in the page, and returns it
-* `dom.removeStyleDeclaration` - Removes a `CSSStyleDeclaration` that was created with `addStyleDeclaration`
+setting styles directly inline on one element. When used, the `addCssRule` returns a handle that allows the rules for the provided selector to be manipulated.
```
Comment (kfranqueiro) [inline README.md]: > When used, the `addCssRule` returns ...

Either drop "the", or add "function" before "returns"

File: README.md [hunk 2]
```diff
@@ -57,7 +52,23 @@ These APIs are implemented in a separate `dom/form` module.
...
+handler to be attached to an element in order to respond to events raised by contained elements that match the provided
+selector.
+
+NOTE: `delegate` will only respond to events raised by contained elements. It will not respond to events raised by the container itself.
+
+
+### Batch Operations
+
+There are many times when a large number of operations need to be applied to the DOM at one time. In order to enable this to be done
+as efficiently as possible, the `schedule` package provides functionality to group the operations together.
```
Comment (kfranqueiro) [inline README.md]: schedule is a module, not a package

**Comment by kfranqueiro** [comment]
Note: This branch was created from the branch used for #34, so merging this would merge that as well.

**Comment by vansimke** [comment]
closing this PR to get a clean branch off of master

**Pull request closed without merging**